openhexa/sdk/pipelines/utils.py

```diff
@@ -131,3 +133,22 @@ def get_local_workspace_config(path: Path):
                         if key != "type":
                             env_vars[stringcase.constcase(f"{slug}_{key.lower()}")] = str(value)
     return env_vars
+
+
+@dataclasses.dataclass
```

Comment:
The dataclasses should probably be in the same module as the `get_pipeline_specs` function.